src/main/controllers/NoAcasNumberController.ts

```diff
@@ -71,11 +70,13 @@ export default class NoAcasNumberController {
   }
 
   public post = (req: AppRequest, res: Response): void => {
+    setUserCaseForRespondent(req, this.form);
     const { saveForLater } = req.body;
+
     if (saveForLater) {
-      handleSaveAsDraft(res);
+      handleSessionErrors(req, res, this.form, PageUrls.CLAIM_SAVED);
+      handleUpdateDraftCase(req, this.logger);
```

Comment:
This line is the same in both branches. Is it worth moving this outside of the if statement entirely?

Reply:
Moved outside if statement